CMakeLists.txt

```diff
@@ -42,23 +45,11 @@ if(TILEDMM_GPU_BACKEND STREQUAL "CUDA")
     list(APPEND TILEDMM_DEFINITIONS TILED_MM_CUDA)
 
 elseif(TILEDMM_GPU_BACKEND STREQUAL "ROCM")
-    find_package(HIPLIBS REQUIRED)
-    find_package(ROCBLAS REQUIRED)
-    list(APPEND TILEDMM_EXTERNAL_LIBRARIES ROCBLAS::rocblas HIPLIBS::hiplibs)
-    list(APPEND TILEDMM_DEFINITIONS -D__HIP_PLATFORM_HCC__) # since we are not using the hcc compiler, we have to set up this explicitly
+    find_package(rocblas REQUIRED)
+    list(APPEND TILEDMM_EXTERNAL_LIBRARIES rocs::rocblas)
```

Comment:
is `rocs` a typo for `rocm` or is it on purpose?

Reply:
Thanks, yes this is a typo.